tests/aio-create.sh

```diff
@@ -177,6 +177,13 @@ function fix_galera_apt_cache {
   popd
 }
 
+function maas_tweaks {
```

Comment:
is this something we should simply check for and add into the `user_extras_secrets.yml` file as we're upgrading?

Reply:
It could probably be a good item to put in the preflight checks before a leapfrog, but in this case, this breaks before we run leapfrog.  I'm guessing that existing environments that have been upgraded to rpc-maas would have ran into this and probably already had the password deployed.

Reply:
Created https://rpc-openstack.atlassian.net/browse/RLM-529 to create a pre-flight check.